src/aria/core/loaders/GeneralTplPreprocessor.js

```diff
@@ -15,7 +15,7 @@
 
 var asyncRequire = require("noder-js/asyncRequire").create(module);
 var firstComment = /^\s*\/\*[\s\S]*?\*\//;
-var alreadyGeneratedRegExp = /^\s*Aria\.classDefinition\(/;
+var alreadyGeneratedRegExp = /^\s*(var\s+|Aria\.classDefinition\()/;
```

Comment:
Even if not changing the behavior in current code state, it is better to clearly define whether the content between parentheses should be a capturing group or not, which means by extension: __do we intend to be able to fetch the corresponding matched content__. If yes, keep it as is. Otherwise, use a non-capturing group (`(?:)`).